v2/internal/runner/enumerate.go

```diff
@@ -152,3 +145,29 @@ func gzipBase64EncodeData(data []byte) string {
 	encoded := base64.StdEncoding.EncodeToString(buf.Bytes())
 	return encoded
 }
+
+func prepareScanListOutput(v nucleicloud.GetScanRequest) output.ListScanOutput {
```

Comment:
instead of eumerate.go, this should go in `cloud.go` along with any other logic for cloud functionality. see my cloud-targets branch for an implementation overview